FixAntenna/NetCore/FixEngine/FixServer.cs

```diff
@@ -31,6 +31,8 @@ namespace Epam.FixAntenna.NetCore.FixEngine
 	/// </summary>
 	public class FixServer
 	{
+        public ITcpListenerFactory TcpListenerFactory {get; set;} = new DefaultTcpListenerFactory();
```

Comment:
Could you use Tab for indentation for all your changes, please?

Reply:
Done :)